### DISCUSSION TITLE
Selecting RefinableStrings when there are many custom managed properties

### DISCUSSION THREAD

**Discussion by sPetsSP** [microsoft-search/pnp-modern-search#3503] [Q&A] [answered]
Hi,

I'm facing a rather weird issue when trying to select the properties of the PnP-Search Results using SharePoint Search. On the tenant there are a lot of custom managed properties and when opening the 'Selected properties' I'm not able to select the RefinableStrings I configured and need.

Anyone encountered this before? Or knows how to get to the RefinableStrings?

![SelectPropertyIssuePnP](https://github.com/microsoft-search/pnp-modern-search/assets/159125766/24dbda97-2d25-4d5f-9488-f0de7a175cde)

Version is 4.1.0.

**Comment by wobba** [comment, marked as answer]
Type it manually into the "Selected properties" field and click ENTER. Then select it.

**Reply by sPetsSP** [reply]
Feel so stupid not to have tried it... thanks!
